File: LeetCode-Problems/Easy/Equivalent-String-Arrays.py
# Python
class Solution(object):
    def arrayStringsAreEqual(self, word1, word2):
        """
        :type word1: List[str]
        :type word2: List[str]
        :rtype: bool
        """
        catWord1 = "";
        catWord2 = "";
        for x in range(0, len(word1)):
            catWord1 += word1[x]
        for x in range (0, len(word2)):
            catWord2 += word2[x]
        if (catWord1 == catWord2):
            return True
        else:
            return False

File: LeetCode-Problems/Easy/Equivalent-String-Arrays.java
//Java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String catWord1 = "";
        String catWord2 = "";
        for (int i = 0; i < word1.length; i++){
            catWord1 += word1[i];
        }
        for (int i = 0; i < word2.length; i++){
            catWord2 += word2[i];
        }
        if (catWord1.equals(catWord2)){
            return true;
        }
        else{
            return false;
        }
    }
}
